### ISSUE TITLE
Empty Relations leading to deleted entities

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#1665]
As documented by Katalin Suba, when en entity is deleted, it still remains in Relations (at least sometimes). See below.
Prevent users from deleting linked entities - not only used in a Statement (which works even now), but even a metaprop, relation, or reference.

So I think this label inclusion turned out to be very useful, because I noticed immediately a pattern:
654f8c04-247a-4ff3-a1ce-b9bb68515abf - "notus erat" problem:
Early on I made a "notorius erat" action, added “notus erat” as SYN, also filled in the SUS, A1S, A2S (you can see this in the entityIds_noexist table). Now this was bad coding, to create an action like that, so with Robert we corrected it later, unlinked it in the statement and deleted it. It seems like the relation links remained, although now they are empty (PrtSc attached).
The same happened with 217dae6b-903f-4a4e-9a27-b077f45b09cc - "predicationem fecit"-case: I made a "predicatio hereticorum", related it with a lot of things in details, but Robert decided that it would be better to use the existing, simple "predicatio" and model the "hereticorum" in properties. So, we deleted it, same as above.
The same case with 40c7cd48-ec1c-4356-b724-78c409e2076b - here I made a new “iudicavit” action, filled in the relations, later deleted it, so same as above.
Of course, I cannot really reconstruct what happened with all the cases, because I cannot figure out what is missing only by looking at the relation labels, if it wasn’t made and deleted later by me and I remember it. But I would say that maybe this pattern applies to others as well. It is also telling that in many cases (16 out of 46) the relation is location identification, and they all come from import, so I guess the identified location got deleted at some point, but the relation links remained there.
To sum it up, I think the problem is that although you cannot delete an entity without unlinking it in the statement, you can delete it with relation links in it, and the relations remain, now leading to nothing. (edited) 
![obrazek](https://user-images.githubusercontent.com/55400794/226307375-4e0d48ad-b78f-4f11-bae4-e34f3dd287ed.png)

**Comment by davidzbiral** [comment]
@adammertel : as part of the solution, also (carefully!) remove the relations leading to non-existing entities. 
You might find of use - talk to Gideon and check before removing the relations whether it is really the case!
https://docs.google.com/spreadsheets/d/19HnGRY9UWe00KVWEdjvCw2uoghT-PhKCtbAViut2MbQ/edit#gid=2078106042

**Comment by jancimertel** [comment]
complementary issue - https://github.com/DISSINET/InkVisitor/issues/1674

**Comment by jancimertel** [comment]
+ removed relations with non-existing entities
